agents/windows/plugins/veeam_backup_status.ps1

```diff
@@ -144,6 +144,28 @@ foreach ($myJob in $myBackupJobs)
 
     }
 
+$myBackupJobs = Get-VBREPJob | where {$_.IsEnabled -eq $true }
+
```

Comment:
I am not sure whether this will work as intended, it looks like there could be a Piggyback section ( `<<<<>>>>` , line 141) interfering?

Even if if that is not the case, I think writing a section header here before entering the loop would be a good idea. It would make the blocks resilient against being moved around, and be more explicit.